docs/ops/pooling/MaxPool_1.md

```diff
@@ -63,24 +63,188 @@
   * **Type**: string
   * **Default value**: *explicit*
   * **Required**: *no*
-  * **Note**: *pads_begin* and *pads_end* attributes are ignored when *auto_pad* is specified.
+  * **Note**: *pads_begin* and *pads_end* attributes are ignored when *auto_pad* is not equal to explicit.
 
 **Inputs**:
 
-*   **1**: 4D or 5D input tensor. Required.
+*   **1**: 3D, 4D or 5D input tensor. Required.
+
+**Outputs**:
+  * **1**: Input shape can be either [N,C,H], [N,C,H,W] or [N,C,H,W,D]. Then the corresponding output shape will be [N,C,H_out], [N,C,H_out,W_out] or [N,C,H_out,W_out,D_out]
```

Comment:
wrap all shapes with back ticks and insert spaces between values for more readability